src/mbgl/renderer/layers/symbol_layer_tweaker.cpp

```diff
@@ -188,16 +187,12 @@ void SymbolLayerTweaker::execute(LayerGroupBase& layerGroup, const PaintParamete
             /*.pitch=*/static_cast<float>(state.getPitch()),
             /*.rotate_symbol=*/rotateInShader,
             /*.aspect_ratio=*/state.getSize().aspectRatio(),
-            /*.pad=*/{0},
+            /*.fade_change=*/parameters.symbolFadeChange,
+            /*.pad=*/0,
         };
 
-        const SymbolDynamicUBO dynamicUBO = {/*.fade_change=*/parameters.symbolFadeChange,
-                                             /*.pad1=*/0,
-                                             /*.pad2=*/{0, 0}};
-
         auto& uniforms = drawable.mutableUniformBuffers();
         uniforms.createOrUpdate(idSymbolDrawableUBOName, &drawableUBO, context);
-        uniforms.createOrUpdate(idSymbolDynamicUBOName, &dynamicUBO, context);
```

Comment:
@TimSylvester This is place where this UBO is update, there is no improvement now.

Reply:
Yeah, looks like it's not even shared.  Seems like perhaps the second half of that effort got lost along the way.  @stefankarschti Are we missing something here?

I'm fine with re-combining them, or moving `DynamicUBO` to a shared UBO and only updating it once per tweaker run.

On Metal, if I can resolve the problems I'm seeing, it won't make much difference.  On GL, a shared approach should lead to fewer `glBufferData` calls, as the dynamic UBO can be updated only once per frame and that keeps the drawable UBO from needing to be updated even when the matrix, etc., hasn't changed.  But that probably benefits the benchmark more than it does for more realistic scenarios.

Reply:
I implemented the `SymbolDynamicUBO` on Metal too, and reverted my old approach. In case we want to merge them we can do them in a separate PR.